### ISSUE TITLE
Add back short substring @: to conveniently find local files by substring

### ISSUE THREAD

**Issue by eugenesvk** [marta-file-manager/marta-issues#934]
Let's say I have a list of files in a folder
```
/.tthidden
/1
/1 (Copy 3)
/1 (Copy)
/2
/2 (Copy 2)
/2 (Copy)
```

Following the tutorial, I invoke a in-folder Look Up action and then try to match:
  - by starting name works (`1` or `2`)
  - by `copy` fails, but then I see in the tutorial that `@:` is to search for substring
  - by `@:copy` also fails (actually `@:1` also fails

What am I doing wrong? Or has tutorial changed?

**Comment by page-down** [comment]
> What am I doing wrong? ...

It works fine for me, press `Cmd+Opt+P` and enter the substring.

Are you trying in a folder path that Spotlight has disabled?
E.g. I won't be able to see any files under `/tmp` via `Look Up`.

**Comment by eugenesvk** [comment]
No, just checked in a `~/Documents`, there is an old (so it's indexed) `_1 copy.pdf` file and I can't find it via `@:copy.pdf` or `copy.pdf`, but can find via the full name `_1 copy.pdf`, but again not via `@:_1 copy.pdf`
So `@:` doesn't seem to be working

**Comment by page-down** [comment]
Oh, I found out why.
The feature description you see refers to `Lookup` in version 0.4.2 (August 2017).
https://marta.sh/blog/look-up/
The new `Lookup` released in 0.8.1 no longer has this functionality.
https://marta.sh/blog/lots-of-updates/

So for now you need to search by `Name contains xxx`.
However it is not yet possible to repeat certain searches by copying and pasting. Because adding a filter relies on **hitting the enter key** after inputting one set of criteria.

For example:

- Enter the `Name contains txt` condition.
- Press Enter.
- Type `prefix_here`.

You can't quickly repeat this search again by pasting `Name contains "txt" prefix_here`.
I think this is very inconvenient, even if it implements the `custom predicate` you mentioned in the other issue.

> ... Or has tutorial changed?

I believe the current version of the tutorial forgot to update to the latest feature description.
Maybe you can update the issue title and content.

**Comment by eugenesvk** [comment]
> So for now you need to search by `Name contains xxx`.

I don't even have this filter `Name contains` in the local lookup `core.lookup.here`, only in the global `core.lookup`

Also, typing `prefix_here` does nothing

But even typing `Name contains "copy" prefix_here` doesn't allow me to find the `_1 copy.pdf` by typing `.pdf` later, I can only find it via the full name, so I need to repeat `Name contains` multiple times, which is indeed an impossible inconvenience

**Comment by LynnWalton** [comment]
I'd love to see something like this added back too.  Its my current biggest frustration with what is otherwise my favorite dual pane file manager for mac.  The way that brings up the dialog is too tedious and doesn't work for me a lot of the time. 
And having the documentation and tutorials match the latest version would be nice too.  But I'm thankful for the software nonetheless.

**Comment by marton78** [comment]
I think substring search should be default! In fact, multiple substrings should even be possible, i.e. `very_long_filename.md` should be found by typing `long name`. This is how search works in other apps with Cmd-P (e.g. VS Code, Zed).
